src/html/partials/story/intro.hbs

```diff
@@ -93,47 +100,47 @@ Sonically, it feels like a much "bigger" song. There are a lot of instruments in
-New genres rarely emerge out of a vacuum - they evolve out of other genres. <i>Love Train</i> probably has more in common with the Philadelphia soul tracks that came before it than with the disco tracks of a few years later. 
+	“Love Train” was arguably the first disco song to top the Hot 100. That alone suggests it ought to stand out, but most of its sonic attributes are in line with the songs of the time.
+</p>
+<p>
+	New genres rarely emerge out of a vacuum&#8202;-&#8202;they evolve out of other genres, and “Love Train” probably has more in common with the Philadelphia Soul tracks that came before it&#8202;-&#8202;themselves an evolution of the Motown sound&#8202;-&#8202;than with the disco tracks of a few years later.
 </p>
 
 <p>
-Notably, the song lacks the high danceability and energy common to most later disco songs. This is especially evident in the relatively milquetoast percussion.
+	Notably, the song lacks the high danceability and energy common to most later disco songs. This is especially evident in the relatively milquetoast percussion.
 </p>
 
 <p>
-The most atypical attributes identified by the algorithm - high instrumentalness and 0 acousticness - are questionable. This isn't an instrumental track (and doesn't even feature any signficant instrumental sections), and while there are clearly some electric guitars involved, it otherwise sounds pretty acoustic, with no traces of the drum machines or synthesizers that were soon to come.
+	The most atypical attributes identified by the algorithm&#8202;-&#8202;high instrumentalness and no acousticness&#8202;-&#8202;are questionable. This isn't an instrumental track (and doesn't even feature any signficant instrumental sections), and while there are clearly some electric guitars involved, it otherwise sounds pretty acoustic, with no traces of the drum machines or synthesizers that were soon to come.
 </p>
 </div>
 
-<div
-  class="step" data-song="The Hustle">
+<div class="step" data-song="The Hustle">
 <p>
-1975's <i>The Hustle</i> amps up the danceability and the energy.
+	By 1975, disco as we know it today had more firmly taken shape. That year’s ’“The Hustle” amped up the danceability and the energy.
 </p>
 
 <p class="step" data-highlight_sonics="instrumentalness">
-It's also notable for being a mostly instrumental track. The only words sung are in the repeated hook "Do the hustle".
+	It's also notable for being a mostly instrumental track. The only words sung in the song are the repeated hook “Do the hustle.”
 </p>
 
-<p
-  class="step" data-song="(Shake, Shake, Shake) Shake Your Booty">
-1976's <i>(Shake, Shake, Shake) Shake Your Booty</i> has a very similar high-danceability, high-energy sonic fingerprint.
+<p class="step" data-song="(Shake, Shake, Shake) Shake Your Booty">
+	1976’s disco hit “(Shake, Shake, Shake) Shake Your ” has a very sonic similar fingerprint, with high danceability and high energy.
```

Comment:
"sonic similar" -> "similar sonic"

(just noting these little fixes so I can remember to make them after merging)